React/tic-tac/src/app/screens/game/components/board/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Square from '../square/index.js';
```

Comment:
I'd recommend moving `Square` to a `components` folder withing the Board component folder.
Also remember that components' folders should be capitalized.